workbench_access.module

```diff
@@ -63,7 +63,11 @@ function workbench_access_form_alter(&$form, FormStateInterface $form_state) {
 function workbench_access_entity_access(EntityInterface $entity, $op, AccountInterface $account) {
   // Return net result of all enabled access schemes. If one scheme allows
   // access, then it is granted.
-  $manager = \Drupal::service('plugin.manager.workbench_access.scheme');
```

Comment:
Part of the point to the service here is to enable issues like https://github.com/agentrickard/workbench_access/issues/122.

IS loading the service really that much of a performance hit?

Reply:
phpstorm is reporting $manager as unused, so I removed it. tests pass so I think its safe to remove the variable

Reply:
Right, but is it called by any other code? We're changing the behavior of the service....

I'm on the road and can look more deeply tomorrow.